### ISSUE TITLE
ToastPopup actionLink on same line as Toast message

### ISSUE THREAD

**Issue by Andy93** [Workday/canvas-kit#678]
## 🐛 Bug Report

Action Link is on the same line as the toast message now and it should be below

## To Reproduce

Steps to reproduce the behavior: Create a toast component with Actionlink and toast message : https://codesandbox.io/s/toast-regression-on-v4-l6cv8?file=/src/index.tsx

## Expected Behavior

Action link to be below the toast message

## Actual Results

Action link is on the same line as the toast message

## Browser (if applicable)

Chrome

## Link to repl or repo (highly encouraged)

Toast component

![Screenshot 2020-05-26 at 15 38 10](https://user-images.githubusercontent.com/1674471/82918289-68b1f980-9f6c-11ea-8687-2bb5dc0e509f.png)

**Comment by mannycarrera4** [comment]
hey @Andy93 thanks for submitting the issue!

Looks like in `v4` we migrated some `type` styles and since the action button extends our `link` type style, it's adding a `display: inline-block` : https://github.com/Workday/canvas-kit/blob/prerelease/v4/modules/core/react/lib/type.ts#L142 resulting in the behavior that you're seeing.

This should be a simple fix to override that display property to `display: block` or `display:flex`

**Comment by alexandrzavalii** [comment]
Thanks for getting back! this is the style for the button.
```
const ActionButton = styled('button')({
  display: 'block',
  backgroundColor: 'transparent', // To prevent Safari from rendering grey 'buttonface' as bgcolor
  border: 'none',
  padding: 0,
  marginTop: spacing.xxxs,
  ...type.body2,
  ...type.variant.link,
});
```
Should we move all the destructuring at the top of the object?

**Comment by mannycarrera4** [comment]
@alexandrzavalii I was just discussing this with ! That might be the approach we take. Still figuring out if  that's what we want to do or use something like `HyperLink` component, however, that changes the design a bit.

**Comment by anicholls** [comment]
Closed by #682
